concent_api/concent_api/celery.py

```diff
@@ -24,7 +24,15 @@
 for feature_name in settings.CONCENT_FEATURES:
 
     app.conf.imports += tuple(
-        ['{}.tasks'.format(app)
+        [f'{app}.tasks'
          for app in AVAILABLE_CONCENT_FEATURES[feature_name]['required_django_apps']
-         if importlib.util.find_spec('{}.tasks'.format(app)) is not None]
+         if importlib.util.find_spec(f'{app}.tasks') is not None]
     )
+
+app.conf.task_create_missing_queues = True
+app.conf.task_routes = ([
```

Comment:
What do you think about moving this to settings or constants ?

Reply:
I wanted to suggest this myself but actually I think I like @dybi's solution more. This is something that we never expect the admin to tweak. Keeping it here as a part of code rather than settings makes it pretty clear.

Reply:
As I wrote above, I like this approach and I think we should leave it as is. `celery.py` seems like a very good place for something like this.